app/controllers/mixins/actions/vm_actions/ownership.rb

```diff
@@ -81,12 +67,13 @@ def get_class_from_controller_param(controller)
         end
 
         # Assign/unassign ownership to a set of objects
-        def ownership(ownership_items = [])
+        def ownership(ownership_ids = [])
           @sb[:explorer] = true if @explorer
           @in_a_form = @ownershipedit = true
           drop_breadcrumb(:name => _("Set Ownership"), :url => "/vm_common/ownership")
-          ownership_items = params[:rec_ids] if params[:rec_ids]
-          build_ownership_info(ownership_items)
+          ownership_ids = params[:rec_ids] if params[:rec_ids]
+          @origin_ownership_items = ownership_ids
```

Comment:
It's not new instance variable, I just change the location where it's set. It's used in `ownership.html.haml`.